README.md

```diff
@@ -18,6 +18,7 @@ programs** with tunable qubit interactions and arbitrary register topologies rea
 [![Documentation](https://github.com/pasqal-io/qadence/actions/workflows/build_docs.yml/badge.svg)](https://pasqal-io.github.io/qadence/latest)
 [![Pypi](https://badge.fury.io/py/qadence.svg)](https://pypi.org/project/qadence/)
 [![License](https://img.shields.io/badge/License-Apache_2.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+![Coverage](./coverage.svg)
```

Comment:
Comparing with the ones above, is there a missing `[` ?

Reply:
Oh right, thanks for pointing that.